.github/workflows/solvcon_runner.yml

```diff
@@ -14,66 +14,29 @@ jobs:
       uses: actions/checkout@v2
       with:
         fetch-depth: 1
+    - name: Set DEVENV_WORKSPACE
+      run: echo "DEVENV_WORKSPACE=${GITHUB_WORKSPACE}" >> $GITHUB_ENV
```

Comment:
See the rationale to use `$GITHUB_ENV` instead of `env context` of Github Action workflow https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/ .